server/src/main/resources/reference.conf

```diff
@@ -1,23 +1,19 @@
 {
-   logger {
-      # This format includes timestamp, level, thread (fiberId), message, and cause
-      format = "%label{timestamp}{%fixed{32}{%timestamp}} %label{level}{%level} %label{thread}{%fiberId} %label{class}{%name:%line} %label{message}{%message} %label{cause}{%cause}"
-      # log filter
-      filter {
-         # rootLevel sets the minimum level of log messages that will be displayed
-         rootLevel = INFO
-      }
-   }
-   postgres {
-      # The JDBC driver class
-      dataSourceClass=org.postgresql.Driver
-      serverName=localhost
-      # set to non-default port, to avoid local collision
-      portNumber=5432
-      databaseName=atum_db
-      user=atum_user
-      password=changeme
-      # maximum number of connections that HikariCP will keep in the pool, including both idle and in-use connections
-      maxPoolSize=10
-   }
+  postgres {
+    # The JDBC driver class
+    dataSourceClass=org.postgresql.Driver
+    serverName=127.0.0.1 // host.docker.internal for local run in docker against db on its host machine; localhost otherwise for testing and for the gh pipeline
+    # set to non-default port, to avoid local collision
```

Comment:
this port is the default PG one (probably a leftover from the past?)

Reply:
yes, will remove that obsolete comment